UnityProject/Assets/Scripts/UI/UI_PressureAlert.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UI_PressureAlert : MonoBehaviour {
+
+    public Sprite[] statusImages; //images to cycle between when active
+	private int activeImageIndex = 0;
+
+	private Image img;
+	private Sprite sprite;
+
+	private int indexLower;
+
+
+	void Start ()
+	{
+		// Cycles the high pressure or low pressure images depending on the value
+
+		img = GetComponent<Image>();
+		sprite = img.sprite;
+		InvokeRepeating("CycleImg", 1f, 1f); //Cycle images every 1 second
```

Comment:
please avoid using reflection-based methods like InvokeRepeating(), use coroutines instead!
or, at **very** least, 
```suggestion
		InvokeRepeating(nameof( CycleImg ), 1f, 1f); //Cycle images every 1 second
```

Reply:
I've changes these to coroutines